crowbar_framework/app/constraints/api_constraint.rb

```diff
@@ -15,13 +15,14 @@
 #
 
 class ApiConstraint
-  attr_reader :version
+  attr_reader :major, :minor
 
   def initialize(options)
-    @version = options.fetch(:version)
+    @major = options.fetch(:major)
+    @minor = options.fetch(:minor)
   end
 
   def matches?(request)
-    request.headers.fetch(:accept) == "application/vnd.crowbar.v#{version}+json"
+    request.headers.fetch(:accept) == "application/vnd.crowbar.v#{major}.#{minor}+json"
```

Comment:
Oh, just `(?<minor>)` doesn't use any external information. It's just a way to tell `match` to extract this capture value into the `:minor` named group. It's not equivalent of `#{minor}` of some sort, that would indeed be bad.

I've just tried, and this worked for me in `pry`:
```ruby
class Foo
  VERSION_MIME= %r(^application/vnd\.crowbar\.v(?<major>\d+).(?<minor>\d+)\+json$)
end

pry(main)> Foo::VERSION_MIME
=> /^application\/vnd\.crowbar\.v(?<major>\d+).(?<minor>\d+)\+json$/
```